packages/connect-analytics/src/types/events.ts

```diff
@@ -27,6 +27,7 @@ export type ConnectAnalyticsEvent =
               referrerApp?: string;
               referrerEmail?: string;
               method?: string;
+              payload?: string[];
```

Comment:
in screenshot, there is `params` and I like it more than `payload`

Reply:
yes, I renamed that later. payload is what use actually passed. params is what connect methods use internally and it might be poluted by some defaults set interally.